CMakeLists.txt

```diff
@@ -249,7 +249,7 @@ add_subdirectory(third_party)
 if(${BUILD_KUZU})
 add_definitions(-DKUZU_ROOT_DIRECTORY="${PROJECT_SOURCE_DIR}")
 add_definitions(-DKUZU_CMAKE_VERSION="${CMAKE_PROJECT_VERSION}")
-add_definitions(-DKUZU_EXTENSION_VERSION="0.3.16")
+add_definitions(-DKUZU_EXTENSION_VERSION="0.3.18")
```

Comment:
Yup thats where I got it from. I bumped the version because the currently version kept on getting overwritten with another branch's changes, but I did not need a rebuild myself, so I switched to a rebuilt version from another branch.